src/core/core.js

```diff
@@ -90,7 +90,12 @@ module.exports = function() {
 				var text = [];
 				text.push('<ul class="' + chart.id + '-legend">');
 				for (var i = 0; i < chart.data.datasets.length; i++) {
-					text.push('<li><span style="background-color:' + chart.data.datasets[i].backgroundColor + '"></span>');
+					if (chart.options.legend.fontFamily) {
+						text.push('<li style="font-family:'+chart.options.legend.fontFamily+';">');
+					} else {
+						text.push('<li style="font-family:'+Chart.defaults.global.legend.labels.fontFamily+';">');
+					}
```

Comment:
Hey, yeah, sorry, I probably could've gone over it a few times. It was just an override function I wrote for some stuff i'm working on at the moment, I ended up changing the code to try and be as similar to the canvas legend as possible (basically we wanted it to look exactly the same as the canvas one but with the ease of hiding and showing based on screen size, because on a mobile everything becomes too squashed) and I just rushed it at the time. But great job on cleaning it up, really appreciate it. I'm not 100% sure why I used chart.options.legend.fontFamily instead of legend.labels.fontFamily (in fact, i'm not even sure how I came to use that, considering the docs even specify to use legend.labels.fontFamily) so that's my bad.